substratevm/mx.substratevm/suite.py

```diff
@@ -300,6 +300,23 @@
             ],
             "workingSets": "SVM",
         },
+        "com.oracle.svm.hosted.jdk11": {
```

Comment:
And we would not need an extra project. We could use one of the existing projects (see `com.oracle.svm.util.ModuleSupport`).